packages/block-editor/src/components/inner-blocks/index.js

```diff
@@ -133,6 +135,7 @@ class InnerBlocks extends Component {
 							rootClientId={ clientId }
 							renderAppender={ renderAppender }
 							__experimentalMoverDirection={ moverDirection }
+							__experimentalConsumeChildToolbar={ consumeChildToolbar }
```

Comment:
Any thoughts about inheritance? E.g: if a block defines  __experimentalConsumeChildToolbars should all the descent blocks have this setting enable or it should only pass one level down if the child does not explicitly set the flag for its descendants?

Reply:
See https://github.com/WordPress/gutenberg/pull/18440#discussion_r346248621